knowledge-base/kb-property-grid-disable-horizontal-scrollbar.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Disable Horizontal Scrollbar
+page_title: How to Hide Horizontal Scrollbar
+description: This article will show you how to disable the horizontal scrollbar of RadPropertyGrid.
+type: how-to
+slug: kb-property-grid-disable-horizontal-scrollbar
+position: 0
+tags: horizontal, scrollbar, propertygrid
+ticketid: 1548960
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td>2022.2 621</td>
+		</tr>
+		<tr>
+			<td>Product</td>
+			<td>RadPropertyGrid for WPF</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+
+Setting the `ScrollViewer.HorizontalScrollBarVisibility` directly onto the `RadPropertyGrid` control instance does not hide it.
```

Comment:
Sounds a bit odd. "Setting the property does not hide it". It is kind of understandable, but make it a bit more clear by saying what is not hidden. "..does not hide the horizontal ScrollBar."